connectors/.gitignore

```diff
@@ -1,2 +1,12 @@
-compiled
-node_modules
\ No newline at end of file
+/node_modules
+
+# misc
+.DS_Store
+*.pem
```

Comment:
Keeping the PEM stuff here because I often end up having some dev pem in my local directory.

Reply:
Same, I actually add a bunch of stuff to `.git/info/exclude` and it acts as a "personal" `.gitignore`
Definitely fine to have that in the shared `.gitignore` though 👍